curiouslearning/FeedTheMonsterJS
# Feat: update build files

# Changes
- update the files inside the build folder

# How to test
-  make sure everything is updated in build folder

Ref: [FM-272](https://curiouslearning.atlassian.net/browse/FM-272)


[FM-272]: https://curiouslearning.atlassian.net/browse/FM-272?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
